server/controllers/users/deactivateUser.js

```diff
@@ -1,3 +1,16 @@
-exports.deactivateUser = (req, res) => {
-  res.send('Hi from deactivate file');
+const { users: { deactivateUser } } = require('../../database/queries');
+
+exports.deactivateUser = (req, res, next) => {
+  // Get the user id and username
+  // const { userId, username } = req.user;
+  // Calling the query
+  deactivateUser(1)
+    .then(({ rows: [{ isactive }] }) => {
+      const response = { id: 1, /* username, */ isactive, message: 'account deactivated' };
```

Comment:
```suggestion
      const response = { id: 1, /* username */ , isactive, data: 'account deactivated' };
```

Reply:
I personally prefer message to data, data is very generic and doesn't give any hints, whereas with message you know you can expect a human-readable string .
With that said, I don't think the response actually needs a message like that, or any response at all for that matter, if the API simply returns HTTP 200 then it means it was successful and client can decide what to display..

Reply:
@11janci that's what I did, I added some info about the user his id and username(which we get fro the request) and I will add status code because as you said it's a common practice